fastlane_bot/utils.py

```diff
@@ -179,15 +183,27 @@ def p_start(self):
     @property
     def p_marg(self):
         if self.y == self.z:
+            # try:
+            new_method = self.decodeRate(self.decodeFloat(int(self.B)) + self.decodeFloat(int(self.A)))
+            # except:
+            #     print(self.B, self.A)
+            #     print(type(self.B), type(self.A))
+            #     print(self.decodeFloat(int(self.B)) + self.decodeFloat(int(self.A)))
+            # print(new_method, self.p_start)
+            assert new_method == self.p_start, f"{new_method}, {self.p_start} **************************************"
             return self.p_start
         elif self.y == 0:
             return self.p_end
```

Comment:
You can get rid of this `elif` part, since it is handled correctly in the `else` part.

Reply:
fixed and cleaned up